src/epoll.rs

```diff
@@ -136,11 +143,12 @@ impl Poller {
     /// Waits for I/O events with an optional timeout.
     #[allow(clippy::needless_update)]
     pub fn wait(&self, events: &mut Events, timeout: Option<Duration>) -> io::Result<()> {
-        log::trace!(
-            "wait: epoll_fd={}, timeout={:?}",
-            self.epoll_fd.as_raw_fd(),
-            timeout
+        let span = tracing::trace_span!(
+            "wait",
+            epoll_fd = ?self.epoll_fd.as_raw_fd(),
+            timeout = ?timeout,
```

Comment:
```suggestion
            ?timeout,
```

(Same as https://github.com/smol-rs/blocking/pull/40#discussion_r1228936512)